config/config-defaults.json

```
@@ -1,5 +1,5 @@
 {
-  "API_SERVER_URL": "https://kubernetes.default.svc",
+  "API_SERVER_URL": "",
```

Cleanup, remove this change.
```suggestion
  "API_SERVER_URL": "https://kubernetes.default.svc",
```